Shared/Samples/Add WFS layer/AddWFSLayerView.swift

```diff
@@ -100,12 +107,12 @@ struct AddWFSLayerView: View {
     
     /// Populates the feature table using queried features contained within a given geometry.
     /// - Parameter geometry: The geometry used to filter the results.
-    private func populateFeatures(within geometry: Geometry?) async throws {
+    private func populateFeatures(within geometry: Geometry) async throws {
```

Comment:
The type here should be envelope as that is what we are always expecting to pass to it, and the name of the parameter should be `extent`.